extensions/vertx-http/deployment/src/test/java/io/quarkus/vertx/http/logging/CustomFormatRequestLoggingHandlerTest.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkus.vertx.http.logging;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Observes;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+import io.restassured.RestAssured;
+import io.vertx.ext.web.Router;
+
+class CustomFormatRequestLoggingHandlerTest {
+
+    private static final String APP_PROPS = "" +
+            "quarkus.http.request-logger.enabled=true\n" +
+            "quarkus.http.request-logger.format=long";
```

Comment:
Can we name them `quarkus.http.log.enabled` and `quarkus.http.log.format` respectively ?